packages/manager/modules/pci/src/projects/new/error/error.routing.js

```diff
@@ -0,0 +1,100 @@
+import get from 'lodash/get';
+import isEmpty from 'lodash/isEmpty';
+import snakeCase from 'lodash/snakeCase';
+
+import EnvironmentService from '@ovh-ux/manager-config';
+
+import { PCI_REDIRECT_URLS } from '../constants';
+import {
+  IMAGE_ASSETS,
+  PAY_DEBT_URL,
+} from './error.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.new.error', {
+      url: '/error',
+      component: 'managerErrorPage',
+      params: {
+        code: {
+          type: 'any',
+        },
+        context: {
+          type: 'any',
+        },
+        detail: null,
+        message: {
+          type: 'any',
+        },
+      },
+      resolve: {
+        breadcrumb: () => null,
+        cancelLink: /* @ngInject */ $state => $state.href('pci'),
+        error: /* @ngInject */ ($transition$, atInternet) => {
+          const page = `public-cloud::${$transition$.to().name.replace(/\./g, '::')}`;
+          const stateParams = $transition$.params();
+          const error = {
+            ...stateParams,
+            code: snakeCase(stateParams.code),
+          };
+
+          atInternet.trackEvent({
+            page,
+            event: `PCI_ERROR_${!isEmpty(error.code) ? error.code.toUpperCase() : 'UNKNOWN'}`,
+          });
+          return error;
+        },
+        image: /* @ngInject */ (error) => {
+          const errorCodeAssets = get(IMAGE_ASSETS, error.code);
+
+          if (errorCodeAssets) {
+            return get(errorCodeAssets, 'src');
+          }
+
+          return get(IMAGE_ASSETS, 'oops.src');
+        },
+        message: /* @ngInject */ ($translate, error) => {
+          switch (error.code) {
+            case 'invalid_payment_mean':
+            case 'paypal_account_not_verified':
+              return $translate.instant(`pci_error_${error.code}`, {
+                href: get(
+                  PCI_REDIRECT_URLS,
+                  `${EnvironmentService.Environment.region}.paymentMethods`,
+                ),
+              });
+            default:
+              return null;
+          }
+        },
+        submitLabel: /* @ngInject */ ($translate, error) => {
+          switch (error.code) {
```

Comment:
This file does a lot of things, maybe you could export to another file instead

Reply:
I agree that some refactor can be done on this, but I think it should be handled separately as the purpose here is to extract an error page component